src/js/components/service-requests/submit-service-request.js

```diff
@@ -102,6 +98,18 @@ export class SubmitServiceRequest {
     this.$element = new FullWidthGrid([$form]).render();
   }
 
+  renderFailure($submitButton) {
+    this.$element.empty().append(
+      new StatusMessage({
+        text: "Error while submitting request.",
+        status: "failure",
+      }).render()
+    );
+    $submitButton
+      .attr({ value: "Submit", disabled: false })
+      .removeClass("button--disabled");
+  }
+
```

Comment:
Depending on your thoughts on the above, this would then be:

```suggestion
  renderFailure() {
    this.$element.empty().append(
      new StatusMessage({
        text: "Error while submitting request.",
        status: "failure",
      }).render()
    );
  }

```